src/gui/optionsdialog.cpp

```diff
@@ -244,6 +244,10 @@ void OptionsDialog::loadBehaviorTabOptions()
 
     initializeStyleCombo();
 
+#ifdef Q_OS_WIN
+    m_ui->labelStyleHint->setText(tr("%1 is recommended for best compatibility with Windows dark mode", "Fusion is recommended for best compatibility with Windows dark mode").arg(u"Fusion"_s));
+#endif
```

Comment:
I would move it in `initializeStyleCombo()` and rename it to `initializeStyleOptions()`